Pipfile

```diff
@@ -9,7 +9,7 @@ name = "pypi"
 
 requests = "*"
 prompt-toolkit = "*"
-python-miio = "*"
+python-miio = ">=0.3.6"
```

Comment:
I guess we should fix it to 0.3.6 to avoid any breakages later and upgrade manually